src/Distributions.jl

```diff
@@ -74,6 +74,7 @@ export
     Biweight,
     Categorical,
     Cauchy,
+    Censored,
```

Comment:
I realized `Truncated` was exported, and we document this distribution, so I added this export.

Reply:
But since other wrappers like `AffineDistribution` are not exported, maybe we don't want to do this? If not, should we even include `Censored` in the docs?

Reply:
`Truncated` is exported for historic reasons I think. It's deprecated and could be removed from the exports in the next breaking release. I would suggest not exporting `Censored`, it also makes it easier to change/optimize the type.

I think it's nevertheless fine to mention it in the docs, it seems useful if users know that a default implementation exists.